src/contracts/core/RewardsCoordinatorStorage.sol

```diff
@@ -89,6 +89,15 @@ abstract contract RewardsCoordinatorStorage is IRewardsCoordinator {
     /// if rewards submission hash for all stakers and operators has been submitted
     mapping(address => mapping(bytes32 => bool)) public isRewardsSubmissionForAllEarnersHash;
 
+    /// @notice Mapping: avs => avsPerformanceRewardsSubmissionHash => bool to check if performance rewards submission hash has been submitted
+    mapping(address => mapping(bytes32 => bool)) public isAVSPerformanceRewardsSubmissionHash;
+
+    /// @notice Mapping: operator => avs => OperatorCommission. The commission an operator takes for a specific AVS.
+    mapping(address => mapping(address => OperatorCommission)) internal operatorAVSCommissionBips;
+
+    /// @notice Mapping: operator => OperatorPICommission. The commission an operator takes for Programmatic Incentives.
+    mapping(address => OperatorCommission) internal operatorPICommissionBips;
+
```

Comment:
I've marked these as `internal` since we'll have separate getters for getting the commission (with defaults included). I don't want these to be directly accessible to other external contracts.